### PR TITLE
Dx 1495

### PR DESCRIPTION
**Pull request by ajrice6713**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-1495.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-1495.s3-website-us-east-1.amazonaws.com/

File: numbers/guides/downloadingReports.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+<BillingReportRetrievalResponse>
+    <ReportStatus>COMPLETED</ReportStatus>
+    <Description>The report archive is constructed.</Description>
+</BillingReportRetrievalResponse>
+```
+{% endextendmethod %}
+
+{% extendmethod %}
+## Download the Report {#download}
+Once the report status query returns `COMPLETED`, we are ready to download the requested report. This can be done performed by making a <code class="get">GET</code> request to the URI received in the location header and appending it with `/file` . The 200 response will include a `Content-Disposition` header indicating the filename and a download of the .zip file will begin.
```
Comment (jmulford-bw) [inline numbers/guides/downloadingReports.md]: Being super nitpicky here, but it'd be nice to clarify that the location header for the 2 requests the user has done so far (POST to create the report, and GET /<id> to check the status) is the same

**Review by jmulford-bw — CHANGES_REQUESTED** [review]

File: numbers/guides/downloadingReports.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+### Request
+```http
+POST https://dashboard.bandwidth.com/api/accounts/{accountId}/billingReports HTTP/1.1
+Content-Type: application/xml; charset=utf-8
+Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
+
+<BillingReport>
+    <Type>BDR</Type>
+    <DateRange>
+        <StartDate>2020-05-21</StartDate>
```
Comment (jchavez443) [inline numbers/guides/downloadingReports.md:47]: Is this the date time format correct?  We really should have one date-time format for our whole api

**Review by jchavez443 — APPROVED** [review]

File: numbers/guides/downloadingReports.md (hunk 2 above)
Comment (ajrice6713) [reply numbers/guides/downloadingReports.md:47]: That is the correct date and time format, which I don't believe matches up with some of the other endpoints

File: numbers/guides/downloadingReports.md (hunk 1 above)
Comment (ajrice6713) [reply numbers/guides/downloadingReports.md]: Nothing wrong with being extra explicit - added some clarification in 53bde3a

**Review by jmulford-bw — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-1495.s3-website-us-east-1.amazonaws.com/

**Pull request merged**